CRM/Case/XMLProcessor/Process.php

```diff
@@ -634,6 +634,10 @@ public function getListeners($caseType) {
    * @return int
    */
   public function getRedactActivityEmail() {
+    $setting = Civi::settings()->get('civicaseRedactActivityEmail');
```

Comment:
The logic for these three method is the same, I wonder would a private function be helpful to remove the duplication?